packages/pangea-sdk/PangeaCyber.Net/AuthZ/AuthZClient.cs

```diff
@@ -0,0 +1,212 @@
+using PangeaCyber.Net.AuthZ.Requests;
+using PangeaCyber.Net.AuthZ.Results;
+using PangeaCyber.Net.Exceptions;
+
+namespace PangeaCyber.Net.AuthZ
+{
+    /// <kind>class</kind>
+    /// <summary>
+    /// AuthZ Client
+    /// </summary>
+    public class AuthZClient : BaseClient<AuthZClient.Builder>
+    {
+        ///
+        public static string ServiceName { get; } = "authz";
+
+        /// Constructor
+        public AuthZClient(Builder builder) : base(builder, ServiceName) { }
+
+        /// <kind>method</kind>
+        /// <summary>
+        /// Create tuples in the AuthZ Service.
+        /// </summary>
+        /// <remarks>Create tuples.</remarks>
+        /// <operationid>authz_post_v1beta_tuple_create</operationid>
+        /// <param name="request" type="PangeaCyber.Net.AuthZ.Requests.TupleCreateRequest">The request to the '/tuple/create' endpoint.</param>
+        /// <returns>Response&lt;TupleCreateResult&gt;</returns>
+        /// <exception cref="PangeaException">Thrown if an error occurs during the operation.</exception>
+        /// <exception cref="PangeaAPIException">Thrown if the API returns an error response.</exception>
+        /// <example>
+        /// <code>
+        /// var request = new TupleCreateRequest
+        /// {
+        ///     new Net.AuthZ.Models.Tuple[] { tuple1, tuple2, tuple3, tuple4 }
+        /// };
+        /// var response = await client.TupleCreate(request);
+        /// </code>
+        /// </example>
+        public async Task<Response<TupleCreateResult>> TupleCreate(TupleCreateRequest request)
```

Comment:
Would be nice to have all these new methods accept `CancellationToken` parameters (check out what I did for `VaultClient.EncryptStructured()`.

Reply:
Done